pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1554,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-time.After(3 * kl.resyncInterval):
```

Comment:
I've just tested it and you're right. What happened is that I was testing it previously with 0.1 instead of 3, so my branch was always running, and the other was not. That's why I haven't noticed it. Thanks for spotting this!